<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QDialog" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strawberry Player</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="artistLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </widget>
  <widget class="QLabel" name="albumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </widget>
  <widget class="QLabel" name="trackLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="genreLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="playcountLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Playcount</string>
   </property>
  </widget>
  <widget class="QLabel" name="songlengthLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Length</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusMessageLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status Messages</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>340</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseButton">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previouseButton">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="finishButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="artist">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="album">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="track">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="year">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="genre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="playcount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="songLength">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusMessage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
